src/mmw/js/src/draw/views.js

```diff
@@ -19,6 +21,17 @@ var $ = require('jquery'),
     placeMarkerTmpl = require('./templates/placeMarker.html'),
     settings = require('../core/settings');
 
+var maxArea = 10000; // 10,000 km^2
+
+function areaWarning(area) {
+    var message = '';
+    message += 'Sorry, your Area of Interest is too large.\n\n';
+    message += Math.floor(area) + ' square km were selected, ';
+    message += 'but the maximum supported size is currently ';
+    message += maxArea + ' square km.';
+    window.alert(message);
```

Comment:
I think we should use a friendlier Modal to display this message, we should have some generic approaches in the codebase.